src/operations/blas1_trees.hpp

```diff
@@ -254,7 +254,7 @@ SYCL_BLAS_INLINE void DoubleAssign<lhs_1_t, lhs_2_t, rhs_1_t, rhs_2_t>::bind(
 template <typename lhs_1_t, typename lhs_2_t, typename rhs_1_t,
           typename rhs_2_t>
 SYCL_BLAS_INLINE void
-DoubleAssign<lhs_1_t, lhs_2_t, rhs_1_t, rhs_2_t>::adjust_access_displacement() {
+DoubleAssign<lhs_1_t, lhs_2_t, rhs_1_t, rhs_2_t>::adjust_access_displacement()  {
```

Comment:
Some of the changes in this file indicate it's not formatted

Reply:
Clang-format appliled